### ISSUE TITLE
Can rmats detect multiple exon skipping event, for example, EGFRviii skipping exon 2-8

### ISSUE THREAD

**Issue by linuxbioinfoguy** [Xinglab/rmats-turbo#309]
Hi, it seem rmats is very good at closeby exon events, how about multiple exon events?

Thanks

**Comment by EricKutschera** [comment]
No, rmats cannot detect splicing events where multiple exons are skipped. The code is only checking for events that fit the structures shown in the diagram from the README: https://github.com/Xinglab/rmats-turbo/tree/v4.1.2#output
